cedar-policy-validator/src/schema/namespace_def.rs

```diff
@@ -499,6 +499,7 @@ impl ValidatorNamespaceDef {
                 }
             }
             if !actions_with_attributes.is_empty() {
+                actions_with_attributes.sort(); // for deterministic error messages
```

Comment:
might be good to reference #833 in this comment

Reply:
I fixed this in other places by using a `BTreeMap`, so maybe use that for uniformity.

Reply:
Added a ref to the issue in the comment since that was the easier "fix."